### DISCUSSION TITLE
2 (conflicting) sets of rules for 1 (mono)repo

### DISCUSSION THREAD

**Discussion by joelgrimberg** [renovatebot/renovate#16429] [Request Help]
my setup:
```
./renovate.json
./package.json
./infrastructure/package.json
```

I have to configure renovate.json that it labels and automerges minor/bugfixes within package.json 🚀

but now I want a different set of rules for the infrastructure/package.json:
I do not want to automerge anything within that infrastructure/package.json. 

is this setup possible within renovate.json?

**Comment by viceice** [comment]
yes.

**Reply by joelgrimberg** [reply]
@viceice thanx for the reply. do you have an example of how I seperate the config 'per folder'

**Reply by viceice** [reply]
check the docs for `matchFiles` and other package rule options

**Comment by PhilipAbed** [comment]
you can add `packageRules`, then under it `matchFiles` and match the file that you want and then you can do whatever rules you want for that specific file match

**Reply by PhilipAbed** [reply]
@joelgrimberg  is this a good example? https://github.com/nownabe/brainfuck-board/blob/654af171d225a0c8418e0986e7ef59bed82af7ce/.github/renovate.json

**Reply by PhilipAbed** [reply]
you can also `matchPaths` i think to upgrade root only.
https://docs.renovatebot.com/configuration-options/#matchpaths
